### PR TITLE
Creosys.OBDAutoDoctor version 3.8.2

### PR DESCRIPTION
**Pull request by EEDDRV**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20040)

### CODE REVIEW

File: manifests/c/Creosys/OBDAutoDoctor/3.8.2/Creosys.OBDAutoDoctor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Creosys.OBDAutoDoctor
+PackageVersion: 3.8.2
+PackageLocale: en-US
+Publisher: Creosys
+PackageName: OBD Auto Doctor
+License: Copyright 2010-2021 Creosys Ltd.
+ShortDescription: OBD Auto Doctor Setup
```
Comment (ImJoakim) [inline manifests/c/Creosys/OBDAutoDoctor/3.8.2/Creosys.OBDAutoDoctor.locale.en-US.yaml:10]:
```
ShortDescription: OBD Auto Doctor Setup
```
Could you please write a better description 👍

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/20040/files#r665137767

File: manifests/c/Creosys/OBDAutoDoctor/3.8.2/Creosys.OBDAutoDoctor.locale.en-US.yaml (hunk 1 above)
Comment (EEDDRV) [reply manifests/c/Creosys/OBDAutoDoctor/3.8.2/Creosys.OBDAutoDoctor.locale.en-US.yaml]: Thank you for your suggestion. I have updated the ShortDescription.

**Review by EEDDRV — COMMENTED** [review]
Have updated the short description.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**